static/src/javascripts/projects/common/modules/experiments/tests/visitor-frequency-quick-survey.js

```diff
@@ -0,0 +1,108 @@
+define([
+    'bean',
+    'fastdom',
+    'lodash/functions/debounce',
+    'common/utils/config',
+    'common/utils/cookies',
+    'common/utils/storage',
+    'common/utils/private-browsing',
+    'text!common/views/experiments/quick-survey.html'
+], function (
+    bean,
+    fastdom,
+    debounce,
+    config,
+    cookies,
+    storage,
+    privateBrowsing,
+    quickSurvey
+) {
+    return function () {
+        this.id = 'VisitorFrequencyQuickSurvey';
+        this.start = '2016-06-13';
+        this.expiry = '2016-06-21';
+        this.author = 'Kate Whalen';
+        this.description = 'Add a single question survey to the submeta section of article pages';
+        this.audience = 0;
+        this.audienceOffset = 0;
+        this.successMeasure = 'Obtain a data-set on how often users visit the Guardian';
+        this.audienceCriteria = 'All users';
+        this.dataLinkNames = 'impressions frequency survey';
+        this.idealOutcome = '';
+
+        this.canRun = function () {
+            return config.page.contentType == 'Article' && checkCookies('GU_FI');
```

Comment:
You could make the cookie name 'GU_FI' a property ie `this.cookieName = 'GU_FI'` then you wouldn't need to pass it as part of `checkCookies` and can use it when you set the cookie here  -> `cookies.add('GU_FI', 1, 5)`